libminifi/test/TestBase.h

```diff
@@ -29,6 +29,13 @@
 #include "core/Core.h"
 #include "properties/Configure.h"
 
+/* Defining the root location of test resource files here in case it moves.   *
```

Comment:
Is this a relic of the original approach?  Don't believe it's currently being used any more.